episodes/01-introduction.md

```diff
@@ -22,28 +20,127 @@ Welcome and code of conduct
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+**Note:** most of the content in this chapter is reproduced or adapted from
+[The Carpentries Instructor Training curriculum](https://carpentries.github.io/instructor-training/01-welcome/index.html).
+
+::: discussion
+
+### Pronouns and Names
+
+Using correct names and pronouns (e.g. "she/her") is important to setting a tone of respect. 
+Learning these is hard to do quickly, so we recommend displaying it prominently during the workshop. 
+
+In an online workshop, give everyone a moment to update their display name to reflect how they would like to be addressed. 
+
+At an in-person event, we recommend supplying name tags and markers, 
+or using plain paper to create table-displayed name placards.
+
+Note that pronouns are personal and some participants might prefer not to share them.
+Do not force people to share their pronouns.
+
+:::
+
+
+## Before The Training Begins
+
+::: challenge
+
+### Getting to know each other
+
+If the Trainer has chosen an
+[icebreaker question](https://carpentries.github.io/instructor-training/icebreakers/index.html),
+participate by writing your answers in the Etherpad.
+
+:::
+
+## Code of Conduct
+
+To make clear what is expected, 
+everyone participating in The Carpentries activities is required to abide by our 
+[Code of Conduct](../CODE_OF_CONDUCT.md). 
+Any form of behaviour to exclude, intimidate, 
+or cause discomfort is a violation of the Code of Conduct. 
+In order to foster a positive and professional learning environment we encourage you to:  
+
+* Use welcoming and inclusive language
+* Be respectful of different viewpoints and experiences
+* Gracefully accept constructive criticism
+* Focus on what is best for the community
+* Show courtesy and respect towards other community members
+
+If you believe someone is violating the Code of Conduct,
+we ask that you report it to The Carpentries Code of Conduct Committee 
+by completing [this form](https://goo.gl/forms/KoUfO53Za3apOuOK2).
+
+::: discussion
+
+### Today's Trainers
+
+To begin class, each Trainer should give a brief introduction of themselves.
+
+(For some guidelines on introducing yourself, see 
+[the _Workshop Introductions_ section of the Instructor Training curriculum](https://carpentries.github.io/instructor-training/23-introductions/index.html)).
+:::
+
+Now, we would like to get to know all of you.
+
 ::::::::::::::::::::::::::::::::::::::  challenge
 
 ## Our First Exercise (10 minutes)
 
-What was the best lesson you ever followed (were taught in a class, read through online, read in a book)?
-Try to differentiate between what was good about the performance of the teacher/trainer
-and what was good about the content of the lesson itself.
+What was the best lesson you ever followed 
+(were taught in a class, read through online, read in a book)?
```

Comment:
```suggestion
What was the best lesson you ever followed ?
This could be a lesson that was taught to you in a class, or one you read through online or in a book.
```

Reply:
Agree ;-)